docs/source/ReferenceGuide.rst

```diff
@@ -361,6 +361,57 @@ The instances should have:
 
 .. [#f1] Avocado plugins can introduce additional test types.
 
+Test execution environment
+--------------------------
+
+Each test is executed in a separate process.  Due to how the
+underlying operating system works, a lot of the attributes of the
+parent process (the Avocado test **runner**) are passed down to the
+test process.
+
+On GNU/Linux systems, a child process should be *"an exact duplicate
+of the parent process, except"* some items that are documented in
+the ``fork(2)`` man page.
+
+Besides those operating system exceptions, the Avocado test runner
+changes the test process in the following ways:
+
+1) The standard input (``STDIN``) is set to a :data:`null device
```

Comment:
This is a sphinx feature called "intersphinx".  It points to Python docs by default for references without a specific prefix.  On my machine, since it's building using Python 2, it points to Python 2 docs.  Let me check if readthedocs.org makes this configurable...  It does, and interestingly enough it's set to "CPython 2.x".  Seems like a RTD issue anyway.